Replace rospublic in astroquery/heasarc/tests/test_heasarc_remote.py

`astroquery/heasarc/tests/test_heasarc_remote.py` has multiple references to `rospublic`. This is no longer a valid HEASARC table after having been deprecated 16 years ago, so all these tests will fail. Replace with `rosmaster` or some other table. The number of rows in the results may need to updated to reflect the query results of the new test table.

Thanks @bsipocz . Works now but there are some failed tests. I'm posting the terminal output here, I'm sorry if it's very long but I wasn't which section of the output might be useful to you so I copied everything

```
python setup.py test -P heasarc -R
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/setup_helpers.py:161: AstropyDeprecationWarning: The package argument to generate_version_py has been deprecated and will be removed in future. Specify the package name in setup.cfg instead
  warnings.warn('The package argument to generate_version_py has '
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/setup_helpers.py:166: AstropyDeprecationWarning: The version argument to generate_version_py has been deprecated and will be removed in future. Specify the version number in setup.cfg instead
  warnings.warn('The version argument to generate_version_py has '
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/setup_helpers.py:171: AstropyDeprecationWarning: The release argument to generate_version_py has been deprecated and will be removed in future. We now use the presence of the "dev" string in the version to determine whether this is a release
  warnings.warn('The release argument to generate_version_py has '
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/setup_helpers.py:186: AstropyDeprecationWarning: Specifying the package name using the "package_name" option in setup.cfg is deprecated - use the "name" option instead.
  warnings.warn('Specifying the package name using the "package_name" '
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/version_helpers.py:239: AstropyDeprecationWarning: The packagename argument to generate_version_py has been deprecated and will be removed in future. Specify the package name in setup.cfg instead
  warnings.warn('The packagename argument to generate_version_py has '
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/version_helpers.py:244: AstropyDeprecationWarning: The version argument to generate_version_py has been deprecated and will be removed in future. Specify the version number in setup.cfg instead
  warnings.warn('The version argument to generate_version_py has '
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/version_helpers.py:249: AstropyDeprecationWarning: The release argument to generate_version_py has been deprecated and will be removed in future. We now use the presence of the "dev" string in the version to determine whether this is a release
  warnings.warn('The release argument to generate_version_py has '
/Users/daisycodes/astroquery/astropy_helpers/astropy_helpers/version_helpers.py:264: AstropyDeprecationWarning: Specifying the package name using the "package_name" option in setup.cfg is deprecated - use the "name" option instead.
  warnings.warn('Specifying the package name using the "package_name" '
running test
installing to temporary directory: /private/var/folders/pb/6yv8xxks74d8svzsrxjvzfx00000gn/T/astroquery-test-avzhx8b3
no previously-included directories found matching 'build'
no previously-included directories found matching 'docs/_build'
no previously-included directories found matching 'docs/api'
no previously-included directories found matching 'astropy_helpers/build'
warning: no previously-included files matching '*.pyc' found anywhere in distribution
warning: no previously-included files matching '*.o' found anywhere in distribution
Searching for pytest-astropy
Best match: pytest-astropy 0.8.0
Processing pytest_astropy-0.8.0-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_astropy-0.8.0-py3.8.egg
Searching for pytest>=4.6
Best match: pytest 6.1.1
Processing pytest-6.1.1-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest-6.1.1-py3.8.egg
Searching for pytest-remotedata>=0.3.1
Best match: pytest-remotedata 0.3.2
Processing pytest_remotedata-0.3.2-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_remotedata-0.3.2-py3.8.egg
Searching for pytest-openfiles>=0.3.1
Best match: pytest-openfiles 0.5.0
Processing pytest_openfiles-0.5.0-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_openfiles-0.5.0-py3.8.egg
Searching for pytest-filter-subpackage>=0.1
Best match: pytest-filter-subpackage 0.1.1
Processing pytest_filter_subpackage-0.1.1-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_filter_subpackage-0.1.1-py3.8.egg
Searching for pytest-doctestplus>=0.2.0
Best match: pytest-doctestplus 0.8.0
Processing pytest_doctestplus-0.8.0-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_doctestplus-0.8.0-py3.8.egg
Searching for pytest-cov>=2.0
Best match: pytest-cov 2.10.1
Processing pytest_cov-2.10.1-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_cov-2.10.1-py3.8.egg
Searching for pytest-astropy-header>=0.1.2
Best match: pytest-astropy-header 0.1.2
Processing pytest_astropy_header-0.1.2-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_astropy_header-0.1.2-py3.8.egg
Searching for pytest-arraydiff>=0.1
Best match: pytest-arraydiff 0.3
Processing pytest_arraydiff-0.3-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pytest_arraydiff-0.3-py3.8.egg
Searching for hypothesis>=5.1
Best match: hypothesis 5.37.1
Processing hypothesis-5.37.1-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/hypothesis-5.37.1-py3.8.egg
Searching for toml
Best match: toml 0.10.1
Processing toml-0.10.1-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/toml-0.10.1-py3.8.egg
Searching for py>=1.8.2
Best match: py 1.9.0
Processing py-1.9.0-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/py-1.9.0-py3.8.egg
Searching for pluggy<1.0,>=0.12
Best match: pluggy 0.13.1
Processing pluggy-0.13.1-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pluggy-0.13.1-py3.8.egg
Searching for packaging
Best match: packaging 20.4
Processing packaging-20.4-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/packaging-20.4-py3.8.egg
Searching for iniconfig
Best match: iniconfig 1.0.1
Processing iniconfig-1.0.1-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/iniconfig-1.0.1-py3.8.egg
Searching for attrs>=17.4.0
Best match: attrs 20.2.0
Processing attrs-20.2.0-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/attrs-20.2.0-py3.8.egg
Searching for psutil
Best match: psutil 5.7.2
Processing psutil-5.7.2-py3.8-macosx-10.14-x86_64.egg

Using /Users/daisycodes/astroquery/.eggs/psutil-5.7.2-py3.8-macosx-10.14-x86_64.egg
Searching for coverage>=4.4
Best match: coverage 5.3
Processing coverage-5.3-py3.8-macosx-10.14-x86_64.egg

Using /Users/daisycodes/astroquery/.eggs/coverage-5.3-py3.8-macosx-10.14-x86_64.egg
Searching for sortedcontainers<3.0.0,>=2.1.0
Best match: sortedcontainers 2.2.2
Processing sortedcontainers-2.2.2-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/sortedcontainers-2.2.2-py3.8.egg
Searching for pyparsing>=2.0.2
Best match: pyparsing 3.0.0a2
Processing pyparsing-3.0.0a2-py3.8.egg

Using /Users/daisycodes/astroquery/.eggs/pyparsing-3.0.0a2-py3.8.egg
WARNING: AstropyDeprecationWarning: The remote_data option should be one of none/astropy/any (found =any). For backward-compatibility, assuming 'any', but you should change the option to be one of the supported ones to avoid issues in future. [astropy.tests.runner]
============================================================================================================ test session starts =============================================================================================================
platform darwin -- Python 3.8.0, pytest-6.1.1, py-1.9.0, pluggy-0.13.1

Running tests with astroquery version 0.4.2.dev6380_testrun.
Running tests in astroquery/heasarc docs/heasarc.

Date: 2020-10-13T07:47:04

Platform: macOS-10.14.6-x86_64-i386-64bit

Executable: /Users/daisycodes/astroquery/venv/bin/python

Full Python Version:
3.8.0 (default, Dec 13 2019, 04:42:36)
[Clang 10.0.1 (clang-1001.0.46.4)]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Package versions:
Numpy: 1.19.2
Matplotlib: not available
Astropy: 4.1rc2
APLpy: not available
pyregion: not available
pyVO: 1.1
astropy-helpers: 4.0.1

Using Astropy options: remote_data: any.

rootdir: /private/var/folders/pb/6yv8xxks74d8svzsrxjvzfx00000gn/T/astroquery-test-avzhx8b3/lib/python3.8/site-packages, configfile: setup.cfg
plugins: filter-subpackage-0.1.1, astropy-header-0.1.2, cov-2.10.1, openfiles-0.5.0, arraydiff-0.3, remotedata-0.3.2, hypothesis-5.37.1, doctestplus-0.8.0
collected 7 items

astroquery/heasarc/tests/test_heasarc_remote.py F.F..F                                                                                                                                                                                 [ 85%]
docs/heasarc/heasarc.rst s                                                                                                                                                                                                             [100%]

================================================================================================================== FAILURES ==================================================================================================================
_______________________________________________________________________________________________________ TestHeasarc.test_basic_example _______________________________________________________________________________________________________

self = <astroquery.heasarc.tests.test_heasarc_remote.TestHeasarc object at 0x11e33f430>

    def test_basic_example(self):
        mission = 'rospublic'
        object_name = '3c273'

        heasarc = Heasarc()
>       table = heasarc.query_object(object_name, mission=mission)

astroquery/heasarc/tests/test_heasarc_remote.py:19:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
astroquery/utils/class_or_instance.py:25: in f
    return self.fn(obj, *args, **kwds)
astroquery/utils/process_asyncs.py:29: in newmethod
    result = self._parse_result(response, verbose=verbose)
astroquery/heasarc/core.py:228: in _parse_result
    table = Table.read(data, hdu=1)
/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/table/connect.py:60: in __call__
    out = registry.read(cls, *args, **kwargs)
/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/io/registry.py:516: in read
    format = _get_valid_format(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

mode = 'read', cls = <class 'astropy.table.table.Table'>, path = None, fileobj = <_io.BytesIO object at 0x11e355090>, args = (<_io.BytesIO object at 0x11e355090>,), kwargs = {'hdu': 1}

    def _get_valid_format(mode, cls, path, fileobj, args, kwargs):
        """
        Returns the first valid format that can be used to read/write the data in
        question.  Mode can be either 'read' or 'write'.
        """

        valid_formats = identify_format(mode, cls, path, fileobj, args, kwargs)

        if len(valid_formats) == 0:
            format_table_str = _get_format_table_str(cls, mode.capitalize())
>           raise IORegistryError("Format could not be identified based on the"
                                  " file name or contents, please provide a"
                                  " 'format' argument.\n"
                                  "The available formats are:\n"
E                                 astropy.io.registry.IORegistryError: Format could not be identified based on the file name or contents, please provide a 'format' argument.
E                                 The available formats are:
E                                            Format           Read Write Auto-identify Deprecated
E                                 --------------------------- ---- ----- ------------- ----------
E                                                       ascii  Yes   Yes            No
E                                                ascii.aastex  Yes   Yes            No
E                                                 ascii.basic  Yes   Yes            No
E                                                   ascii.cds  Yes    No            No
E                                      ascii.commented_header  Yes   Yes            No
E                                                   ascii.csv  Yes   Yes           Yes
E                                               ascii.daophot  Yes    No            No
E                                                  ascii.ecsv  Yes   Yes           Yes
E                                            ascii.fast_basic  Yes   Yes            No
E                                 ascii.fast_commented_header  Yes   Yes            No
E                                              ascii.fast_csv  Yes   Yes            No
E                                        ascii.fast_no_header  Yes   Yes            No
E                                              ascii.fast_rdb  Yes   Yes            No
E                                              ascii.fast_tab  Yes   Yes            No
E                                           ascii.fixed_width  Yes   Yes            No
E                                 ascii.fixed_width_no_header  Yes   Yes            No
E                                  ascii.fixed_width_two_line  Yes   Yes            No
E                                                  ascii.html  Yes   Yes           Yes
E                                                  ascii.ipac  Yes   Yes            No
E                                                 ascii.latex  Yes   Yes           Yes
E                                             ascii.no_header  Yes   Yes            No
E                                                   ascii.rdb  Yes   Yes           Yes
E                                                   ascii.rst  Yes   Yes            No
E                                            ascii.sextractor  Yes    No            No
E                                                   ascii.tab  Yes   Yes            No
E                                                        asdf  Yes   Yes           Yes
E                                                        fits  Yes   Yes           Yes
E                                                        hdf5  Yes   Yes           Yes
E                                                  pandas.csv  Yes   Yes            No
E                                                  pandas.fwf  Yes    No            No
E                                                 pandas.html  Yes   Yes            No
E                                                 pandas.json  Yes   Yes            No
E                                                     votable  Yes   Yes           Yes
E                                                      aastex  Yes   Yes            No        Yes
E                                                         cds  Yes    No            No        Yes
E                                                         csv  Yes   Yes            No        Yes
E                                                     daophot  Yes    No            No        Yes
E                                                        html  Yes   Yes            No        Yes
E                                                        ipac  Yes   Yes            No        Yes
E                                                       latex  Yes   Yes            No        Yes
E                                                         rdb  Yes   Yes            No        Yes

/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/io/registry.py:596: IORegistryError
_______________________________________________________________________________________________________ TestHeasarc.test_mission_cols ________________________________________________________________________________________________________

self = <astroquery.heasarc.tests.test_heasarc_remote.TestHeasarc object at 0x11e4f4160>

    def test_mission_cols(self):
        heasarc = Heasarc()
        mission = 'rospublic'
>       cols = heasarc.query_mission_cols(mission=mission)

astroquery/heasarc/tests/test_heasarc_remote.py:34:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
astroquery/heasarc/core.py:103: in query_mission_cols
    return self._parse_result(response).colnames
astroquery/heasarc/core.py:228: in _parse_result
    table = Table.read(data, hdu=1)
/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/table/connect.py:60: in __call__
    out = registry.read(cls, *args, **kwargs)
/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/io/registry.py:516: in read
    format = _get_valid_format(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

mode = 'read', cls = <class 'astropy.table.table.Table'>, path = None, fileobj = <_io.BytesIO object at 0x10dfddcc0>, args = (<_io.BytesIO object at 0x10dfddcc0>,), kwargs = {'hdu': 1}

    def _get_valid_format(mode, cls, path, fileobj, args, kwargs):
        """
        Returns the first valid format that can be used to read/write the data in
        question.  Mode can be either 'read' or 'write'.
        """

        valid_formats = identify_format(mode, cls, path, fileobj, args, kwargs)

        if len(valid_formats) == 0:
            format_table_str = _get_format_table_str(cls, mode.capitalize())
>           raise IORegistryError("Format could not be identified based on the"
                                  " file name or contents, please provide a"
                                  " 'format' argument.\n"
                                  "The available formats are:\n"
E                                 astropy.io.registry.IORegistryError: Format could not be identified based on the file name or contents, please provide a 'format' argument.
E                                 The available formats are:
E                                            Format           Read Write Auto-identify Deprecated
E                                 --------------------------- ---- ----- ------------- ----------
E                                                       ascii  Yes   Yes            No
E                                                ascii.aastex  Yes   Yes            No
E                                                 ascii.basic  Yes   Yes            No
E                                                   ascii.cds  Yes    No            No
E                                      ascii.commented_header  Yes   Yes            No
E                                                   ascii.csv  Yes   Yes           Yes
E                                               ascii.daophot  Yes    No            No
E                                                  ascii.ecsv  Yes   Yes           Yes
E                                            ascii.fast_basic  Yes   Yes            No
E                                 ascii.fast_commented_header  Yes   Yes            No
E                                              ascii.fast_csv  Yes   Yes            No
E                                        ascii.fast_no_header  Yes   Yes            No
E                                              ascii.fast_rdb  Yes   Yes            No
E                                              ascii.fast_tab  Yes   Yes            No
E                                           ascii.fixed_width  Yes   Yes            No
E                                 ascii.fixed_width_no_header  Yes   Yes            No
E                                  ascii.fixed_width_two_line  Yes   Yes            No
E                                                  ascii.html  Yes   Yes           Yes
E                                                  ascii.ipac  Yes   Yes            No
E                                                 ascii.latex  Yes   Yes           Yes
E                                             ascii.no_header  Yes   Yes            No
E                                                   ascii.rdb  Yes   Yes           Yes
E                                                   ascii.rst  Yes   Yes            No
E                                            ascii.sextractor  Yes    No            No
E                                                   ascii.tab  Yes   Yes            No
E                                                        asdf  Yes   Yes           Yes
E                                                        fits  Yes   Yes           Yes
E                                                        hdf5  Yes   Yes           Yes
E                                                  pandas.csv  Yes   Yes            No
E                                                  pandas.fwf  Yes    No            No
E                                                 pandas.html  Yes   Yes            No
E                                                 pandas.json  Yes   Yes            No
E                                                     votable  Yes   Yes           Yes
E                                                      aastex  Yes   Yes            No        Yes
E                                                         cds  Yes    No            No        Yes
E                                                         csv  Yes   Yes            No        Yes
E                                                     daophot  Yes    No            No        Yes
E                                                        html  Yes   Yes            No        Yes
E                                                        ipac  Yes   Yes            No        Yes
E                                                       latex  Yes   Yes            No        Yes
E                                                         rdb  Yes   Yes            No        Yes

/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/io/registry.py:596: IORegistryError
_______________________________________________________________________________________________________ TestHeasarc.test_query_region ________________________________________________________________________________________________________

self = <astroquery.heasarc.tests.test_heasarc_remote.TestHeasarc object at 0x11fe8b400>

    def test_query_region(self):
        heasarc = Heasarc()
        mission = 'rospublic'

        # Define coordinates for '3c273' object
        c = commons.coord.SkyCoord('12h29m06.70s +02d03m08.7s', frame='icrs')
>       table = heasarc.query_region(c, mission=mission, radius='1 degree')

astroquery/heasarc/tests/test_heasarc_remote.py:68:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
astroquery/utils/class_or_instance.py:25: in f
    return self.fn(obj, *args, **kwds)
astroquery/utils/process_asyncs.py:29: in newmethod
    result = self._parse_result(response, verbose=verbose)
astroquery/heasarc/core.py:228: in _parse_result
    table = Table.read(data, hdu=1)
/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/table/connect.py:60: in __call__
    out = registry.read(cls, *args, **kwargs)
/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/io/registry.py:516: in read
    format = _get_valid_format(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

mode = 'read', cls = <class 'astropy.table.table.Table'>, path = None, fileobj = <_io.BytesIO object at 0x11e5a7e00>, args = (<_io.BytesIO object at 0x11e5a7e00>,), kwargs = {'hdu': 1}

    def _get_valid_format(mode, cls, path, fileobj, args, kwargs):
        """
        Returns the first valid format that can be used to read/write the data in
        question.  Mode can be either 'read' or 'write'.
        """

        valid_formats = identify_format(mode, cls, path, fileobj, args, kwargs)

        if len(valid_formats) == 0:
            format_table_str = _get_format_table_str(cls, mode.capitalize())
>           raise IORegistryError("Format could not be identified based on the"
                                  " file name or contents, please provide a"
                                  " 'format' argument.\n"
                                  "The available formats are:\n"
E                                 astropy.io.registry.IORegistryError: Format could not be identified based on the file name or contents, please provide a 'format' argument.
E                                 The available formats are:
E                                            Format           Read Write Auto-identify Deprecated
E                                 --------------------------- ---- ----- ------------- ----------
E                                                       ascii  Yes   Yes            No
E                                                ascii.aastex  Yes   Yes            No
E                                                 ascii.basic  Yes   Yes            No
E                                                   ascii.cds  Yes    No            No
E                                      ascii.commented_header  Yes   Yes            No
E                                                   ascii.csv  Yes   Yes           Yes
E                                               ascii.daophot  Yes    No            No
E                                                  ascii.ecsv  Yes   Yes           Yes
E                                            ascii.fast_basic  Yes   Yes            No
E                                 ascii.fast_commented_header  Yes   Yes            No
E                                              ascii.fast_csv  Yes   Yes            No
E                                        ascii.fast_no_header  Yes   Yes            No
E                                              ascii.fast_rdb  Yes   Yes            No
E                                              ascii.fast_tab  Yes   Yes            No
E                                           ascii.fixed_width  Yes   Yes            No
E                                 ascii.fixed_width_no_header  Yes   Yes            No
E                                  ascii.fixed_width_two_line  Yes   Yes            No
E                                                  ascii.html  Yes   Yes           Yes
E                                                  ascii.ipac  Yes   Yes            No
E                                                 ascii.latex  Yes   Yes           Yes
E                                             ascii.no_header  Yes   Yes            No
E                                                   ascii.rdb  Yes   Yes           Yes
E                                                   ascii.rst  Yes   Yes            No
E                                            ascii.sextractor  Yes    No            No
E                                                   ascii.tab  Yes   Yes            No
E                                                        asdf  Yes   Yes           Yes
E                                                        fits  Yes   Yes           Yes
E                                                        hdf5  Yes   Yes           Yes
E                                                  pandas.csv  Yes   Yes            No
E                                                  pandas.fwf  Yes    No            No
E                                                 pandas.html  Yes   Yes            No
E                                                 pandas.json  Yes   Yes            No
E                                                     votable  Yes   Yes           Yes
E                                                      aastex  Yes   Yes            No        Yes
E                                                         cds  Yes    No            No        Yes
E                                                         csv  Yes   Yes            No        Yes
E                                                     daophot  Yes    No            No        Yes
E                                                        html  Yes   Yes            No        Yes
E                                                        ipac  Yes   Yes            No        Yes
E                                                       latex  Yes   Yes            No        Yes
E                                                         rdb  Yes   Yes            No        Yes

/Users/daisycodes/astroquery/venv/lib/python3.8/site-packages/astropy-4.1rc2-py3.8-macosx-10.14-x86_64.egg/astropy/io/registry.py:596: IORegistryError
========================================================================================================== short test summary info ===========================================================================================================
FAILED astroquery/heasarc/tests/test_heasarc_remote.py::TestHeasarc::test_basic_example - astropy.io.registry.IORegistryError: Format could not be identified based on the file name or contents, please provide a 'format' argument.
FAILED astroquery/heasarc/tests/test_heasarc_remote.py::TestHeasarc::test_mission_cols - astropy.io.registry.IORegistryError: Format could not be identified based on the file name or contents, please provide a 'format' argument.
FAILED astroquery/heasarc/tests/test_heasarc_remote.py::TestHeasarc::test_query_region - astropy.io.registry.IORegistryError: Format could not be identified based on the file name or contents, please provide a 'format' argument.
================================================================================================== 3 failed, 3 passed, 1 skipped in 12.59s ===================================================================================================
```

--- a/astroquery/heasarc/tests/test_heasarc_remote.py
+++ b/astroquery/heasarc/tests/test_heasarc_remote.py
@@ -12,7 +12,7 @@ from ...utils import commons
 class TestHeasarc:
 
     def test_basic_example(self):
-        mission = 'rospublic'
+        mission = 'rosmaster'
         object_name = '3c273'
 
         heasarc = Heasarc()
@@ -30,10 +30,10 @@ class TestHeasarc:
 
     def test_mission_cols(self):
         heasarc = Heasarc()
-        mission = 'rospublic'
+        mission = 'rosmaster'
         cols = heasarc.query_mission_cols(mission=mission)
 
-        assert len(cols) == 28
+        assert len(cols) == 29
 
         # Test that the cols list contains known names
         assert 'EXPOSURE' in cols
@@ -42,7 +42,7 @@ class TestHeasarc:
         assert 'SEARCH_OFFSET_' in cols
 
     def test_query_object_async(self):
-        mission = 'rospublic'
+        mission = 'rosmaster'
         object_name = '3c273'
 
         heasarc = Heasarc()
@@ -52,7 +52,7 @@ class TestHeasarc:
 
     def test_query_region_async(self):
         heasarc = Heasarc()
-        mission = 'rospublic'
+        mission = 'rosmaster'
         c = commons.coord.SkyCoord('12h29m06.70s +02d03m08.7s', frame='icrs')
         response = heasarc.query_region_async(c, mission=mission,
                                               radius='1 degree')
@@ -61,7 +61,7 @@ class TestHeasarc:
 
     def test_query_region(self):
         heasarc = Heasarc()
-        mission = 'rospublic'
+        mission = 'rosmaster'
 
         # Define coordinates for '3c273' object
         c = commons.coord.SkyCoord('12h29m06.70s +02d03m08.7s', frame='icrs')

